src/psalter.php

```diff
@@ -302,6 +306,20 @@ function (string $line) : bool {
     }
 }
 
+if (isset($options['allow-backwards-incompatible-changes'])) {
+    $allow_backwards_incompatible_changes = filter_var(
+        $options['allow-backwards-incompatible-changes'],
+        FILTER_VALIDATE_BOOLEAN,
+        ['flags' => FILTER_NULL_ON_FAILURE]
+    );
+
+    if ($allow_backwards_incompatible_changes === null) {
+        die('--allow-backwards-incompatible-changes expectes a boolean value [true|false|1|0]' . PHP_EOL);
```

Comment:
This gives the user a choice of five options - the four listed above, plus not passing the option at all. Only two options are really needed, and in general the fewer options we give people the easier it should be for them to make good decisions.

Reply:
Yeah, this was my suggestion but I'd also be happy with `--prevent-backwards-incompatible-changes`, too

Reply:
I prefer not to negate boolean variables (or flags as in this case). 'incompatible' is already the negated form of 'compatible' so that would make it a double negation. But I'll let you decide @muglug.

Reply:
Yeah, it's generally seen as bad design (hence why I have `--show-info=false` vs `--hide-info`). On the other hand, it's less wordy.

Your PR, you choose (and it sounds like you want to stick with the existing one)

Reply:
Say the word, and I'll merge

Reply:
Let's keep it this way to stay consistent with the other flags. Thanks @bdsl for the suggestion though!